### ISSUE TITLE
Accessible Poster

### ISSUE THREAD

**Issue by yiwenteng** [IITIDIDX597/F2023_Team1#34]
GOAL:
For the UChicago Medicine HR team who wants to increase registration rates for the MyWellbeing employee wellness program by 24%, the MWB communication toolkit that has all the engagement components to increase awareness + staff engagement with MWB so that more employees can engage with the health resources they need to be as healthy as possible.

HILL:
UCM employees with limited opportunity to check email during their workday are able to learn about MWB in a manner that is convenient for them so that they can utilize ALL the free wellness resources they are entitled to.

SUBHILL:
Create a physical MWB marketing campaign specifically for employees with limited opportunity to check email.

User story:
As someone who wears glasses, 
I want the poster text to be large enough 
So that I can read it from 4 feet away.

Acceptance criteria: 
Given that... I cannot read small text from far away
When.. I see a MWB poster in the hallway
Then.. the text needs to be at least size 30 font so that I can easily read it while walking by.

**Comment by akallish** [comment]
@yiwenteng this is great and specific. Look up on google how large type has to be from 4 feet away to accurately read a headline. I would say in a kitchen the poster maybe 8 feet away!

**Comment by yiwenteng** [comment]
Criteria: 
Given that I need to read text from at least 8 feet away in the kitchen environment,
When I look at an MWB poster,
Then the headline text should be a minimum of size 48 font (which is readable from 8 feet away according to standard visibility guidelines), ensuring that I can easily read and understand the main message as I pass by.

**Comment by akallish** [comment]
@yiwenteng This story is about reading the poster.

I like the increased specificity.

Also, am not sure but this acceptance criteria and story should be in the original story issue so I can se the goal, hill statement and sub-hills for tracibility.

**Comment by jieanjie** [comment]
User Story:
As a person who wears glasses, I want the text on MWB posters to be large enough so that I can easily read it from a distance of 4 feet.

Acceptance Criteria:

Given that I struggle to read small text from afar,
When I see an MWB poster in the hallway,
Then the text on the poster should be at least size 30 font, allowing me to comfortably read it while passing by.

**Comment by akallish** [comment]
@jieanjie This may be too simple as @yiwenteng has more informational richness in the acceptance criteria

**Comment by lidajiang19** [comment]
User story:
As someone who wears glasses,
I want the poster text to be large enough
So that I can read it from 4 feet away.

A/C:
Given that I am not good at reading small text,
When I noticed a MWB poster in the hallway,
Then the text should be at least size 48, enabling me to read most important information at first sight.

**Comment by emerydonovan** [comment]
these iterations make sense to me. They also make me realize that legibility from far away matters for both glasses wearers and just anyone walking by - not sure if it's worth expanding who the user is if it's the same feature?

**Comment by akallish** [comment]
@emerydonovan good point. I do not believe there are clear legibility standards for type by distance other than federal traffic signage standards. 

People with glasses for farsightedness will read 20/20

**Comment by jieanjie** [comment]
Final:
Story:
As someone who wears glasses,
I want the poster text to be large enough
So that I can read it from 4 feet away.
 
A/C
Given that I cannot read small text from far away
When I see a MWB poster in the hallway
Then the text needs to be at least size 30 font so that I can easily read it while walking by.
